src/api/Sources.js

```diff
@@ -256,5 +256,5 @@ export default function getSourceInfo() {
       monarchUsage: "Monarch stores gene to phenotype associations and gene function data",
       vocabulary: "APO, RO"
     }
-  ];
+  ]
```

Comment:
The linter complains because we're using it as javascript even though it's mostly just json, but either way is fine

Reply:
Thanks Kent - what link error are you seeing? This seems to pass my linter:

```
$ vue-cli-service lint --no-fix src/api/Sources.js 

DONE  No lint errors found!
```

(Also what linter should I be using?)

Reply:
odd, I get:

    error: Missing semicolon (semi) at src/api/Sources.js:259:4:
      257 |       vocabulary: "APO, RO"
      258 |     }
    > 259 |   ]
          |    ^
      260 | }
      261 | 

I'm running npm run lint which calls vue-cli-service lint --no-fix, so should be the same, node version 8.12.0

Reply:
Oh, sorry!  I saw that error too, fixed it, and forgot (thus no error for me). Should be linted now if you do a pull